<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QMainWindow" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="registration_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
   </widget>
   <widget class="QLabel" name="company_name_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Company name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="company_name_line">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="email_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_line">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="phone_number_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phone_number_line">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="address_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_line">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="password_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_line">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirm_password_line">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="confirm_password_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="invalid_email_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>✓</string>
    </property>
   </widget>
   <widget class="QLabel" name="invalid_phone_number_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>✓</string>
    </property>
   </widget>
   <widget class="QLabel" name="invalid_address_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>✓</string>
    </property>
   </widget>
   <widget class="QLabel" name="invalid_password_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>✓</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwords_match_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>✓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="authorisation_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Back to authorisation</string>
    </property>
   </widget>
   <widget class="QLabel" name="invalid_company_name_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>✓</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
